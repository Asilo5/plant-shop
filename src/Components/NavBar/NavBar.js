import React from 'react';

import Search from '../../Containers/Search/Search';

const NavBar  = () => {

    return (
        <nav>
           <h1>Plant Shop</h1>

           <div>
               <Search />
               <p>Add Plant</p>
               <p>Loved</p>
           </div>
        </nav>
    )
}

export default NavBar;