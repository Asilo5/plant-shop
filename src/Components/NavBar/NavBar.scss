nav {
    display: flex;
    justify-content: space-between;
    padding: 20px;
    h1 {
      font-size: 2em;
    }
    div {
        display: flex;
        .nav {
            padding: 10px;
            font-size: 20px;
            text-decoration: none;
            color: black;
            margin-right: 20px;
        }
        input {
            font-size: 20px;
            background-color: transparent;
            border: none;
            border-bottom: 2px solid black;
            margin-right: 25px;
        }
        button {
            font-size: 20px;
            background-color: transparent;
            border: none;
            padding-right: 10px;
            transition: all .2s ease-in-out;
            &:hover {
                transform: scale(1.2);
                cursor: pointer;
            }
        }
    }
}