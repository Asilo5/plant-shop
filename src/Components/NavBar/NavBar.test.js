import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';

describe('NavBar component', () => {
  
  let wrapper;

  beforeEach(() => {
      wrapper = shallow(<NavBar />)
  })

  it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
  })

})