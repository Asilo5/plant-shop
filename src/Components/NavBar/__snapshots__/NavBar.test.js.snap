// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component should match snapshot 1`] = `
<nav>
  <NavLink
    className="logo"
    to="/"
  >
    Plant Shop
  </NavLink>
  <div>
    <Search />
    <NavLink
      className="nav"
      to="/addPlant"
    >
      Add Plant 
    </NavLink>
    <NavLink
      className="nav"
      to="/lovedPlant"
    >
      Loved Plants
    </NavLink>
  </div>
</nav>
`;
